package.json

```diff
@@ -31,6 +31,7 @@
     "@angular/router": "~9.0.2",
     "@ngneat/transloco": "^2.16.0",
     "@nguniversal/express-engine": "^9.0.2",
+    "@types/jsdom": "^12.2.4",
```

Comment:
I think we dont need this in the main package json as the error is when executing the tsc command inside functions